### ISSUE TITLE
Rebase theme on Adwaita?

### ISSUE THREAD

**Issue by karaign** [trgeiger/pop-kde#2]
Since the new version of Pop!_OS has switched to an Adwaita-based theme it would be nice if the Kvantum theme was updated to reflect that.

**Comment by trgeiger** [comment]
That's a great idea. I haven't used KDE for a little while now but since users like you are using this I could spruce it up soon!



Tayler Geiger


---- On Mon, 04 May 2020 19:49:13 -0500 notifications@github.com wrote ----



Since the new version of Pop!_OS has switched to an Adwaita-based theme it would be nice if the Kvantum theme was updated to reflect that.

**Comment by Vistaus** [comment]
Yeah, it would be great to have a new version of this theme. I thought the old one was so-so (the official one, I mean), but the current version of the Pop!_OS theme is very, very nice! Would love to use it on my system since I only use Qt-based DE's.
